### ISSUE TITLE
ionic 4 ionic infinite scroll event not being triggered..this is sick as it was working few days ago 

### ISSUE THREAD

**Issue by frank505** [ionic-team/ionic#18673]
# Bug Report

**Ionic version:**

[x] **4.x**

**Current behavior:**

**Expected behavior:**

**Steps to reproduce:**

**Related code:**

```
insert short code snippets here
```

**Other information:**

**Ionic info:**

```
insert the output from ionic info here
```

**Comment by frank505** [comment]
```
  <ion-infinite-scroll threshold="25%"  (ionInfinite)="LoadMoreUserDataBasedOnCurrentSearchInputValue($event)">
      <ion-infinite-scroll-content
        loadingSpinner="crescent"
        loadingText="Loading more data...">
      </ion-infinite-scroll-content>
    </ion-infinite-scroll>
```
and here is the function that handles this event

```
LoadMoreUserDataBasedOnCurrentSearchInputValue(event)
  {
    if(this.search_input_value=="" || this.search_input_value==null){
      this.LoadMoreData(event);
    }else{
      this.LoadMoreSearchedContent(event);
    }
  }
```
 honestly it woked few days ago until my hard drive crashed and after i fixed it infinitescroll no longer works any help will be appreciated..

**Comment by tanoabeleyra** [comment]
Can confirm, I started a new project and it isn't working (Ionic 4.6.0). Downgraded to Ionic 4.5.0 and it works.

**Comment by rodolfofranco** [comment]
Same behaviour, here's my `ionic info` output:

Ionic:

   Ionic CLI                     : 5.2.0 (/home/rodolfo/.nvm/versions/node/v11.14.0/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.6.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Capacitor:

   Capacitor CLI   : 1.1.0
   @capacitor/core : 1.1.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : none
   Cordova Plugins   : no whitelisted plugins (1 plugins total)

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v11.14.0 (/home/rodolfo/.nvm/versions/node/v11.14.0/bin/node)
   npm    : 6.9.0
   OS     : Linux 4.18

**Comment by tanoabeleyra** [comment]
This should be closed in favor of #18632

**Comment by liamdebeasi** [comment]
Thanks for the issue! I am going to close this as a duplicate of https://github.com/ionic-team/ionic/issues/18632.

We have a fix in place, and it will be released as part of 4.6.1.

Thanks!

**Comment by frank505** [comment]
honestly i downgraded to 4.5.0 created a new ionic project and still the infinite scroll doesnt....i really dont know why.......i went ahead to copy the exact code on ionics official docs and added the exact code for angular and still infinite scroll failed to load i am stuck right now any advices would be appreciated
